artemis-commons/src/test/java/org/apache/activemq/artemis/utils/NetworkHealthTest.java

```diff
@@ -179,6 +182,7 @@ private void doCheck(String localaddress) throws Exception {
 
    }
 
+   @Ignore("doesn't work in Travis CI environment")
```

Comment:
Is this because its failing due to lack of IPV6 in the env? If the test really needs that it should probably check it works and fail an assumption which will make the test skip in envs it cant possibly work in, as opposed to disabling the test everywhere.

Reply:
From what I can tell the failure is due to the way the ping command is being constructed and executed in org.apache.activemq.artemis.core.server.NetworkHealthCheck#purePing (i.e. via java.lang.ProcessBuilder). It appears the Travis CI environment simply won't allow it.  I'll investigate creating an assumption to test the environment.